### PR TITLE
add 26. Remove Duplicates from Sorted Array

### PR DESCRIPTION
**Pull request by katsurada1**
Please check my code.

### CODE REVIEW

File: 26.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        """A function to remove the duplicates in-place
+        Args:
+            nums (List[int]): A given an integer array nums sorted in non-decreasing order.
+        Returns:
+            int: The elements after removing the duplicates.
+        """
+        n = 0
+        for i in range(len(nums)):
+            if(nums[n] < nums[i]):
+                n+=1
```
Comment (kobakobashu) [inline 26.py:16]: n += 1

File: 26.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+                nums[n]=nums[i]
```
Comment (kobakobashu) [inline 26.py:17]: nums[n] = nums[i]

File: 26.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        return(n+1)
```
Comment (kobakobashu) [inline 26.py:18]: return n + 1

**Review by kobakobashu — CHANGES_REQUESTED** [review]
👀

**Comment by katsurada1** [comment]
I modified the code.
Please check my code.

**Comment by kobakobashu** [comment]
LGTM.
Please delete 21.py and merge only the 26.py

**Review by kobakobashu — APPROVED** [review]

**Pull request merged**